setup.py

```diff
@@ -156,7 +156,7 @@ def run(self):
         prefix = os.path.abspath(self.prefix)
         self.set_data_files(prefix)
         self.do_egg_install()
-        self.ooniresources()
+        #self.ooniresources()
```

Comment:
@dantheta This change cannot be merged into master. I see that this was already discussed in #395. I agree that adding a command line flag to disable downloading ooni resources is a good solution. Otherwise, I'd say, just remove this part of the diff and keep your own fork with this (and possibly other) minimal patches to ooni probe.